go.mod

```
@@ -125,6 +125,11 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
+	github.com/aws/aws-sdk-go v1.44.116
+	github.com/spf13/cobra v1.6.0
+)
```

Can we merge this in with the main require section?
```suggestion
	github.com/aws/aws-sdk-go v1.44.116
	github.com/spf13/cobra v1.6.0
)
```